plugins/woocommerce/client/admin/client/settings-email/style.scss

```diff
@@ -111,3 +111,61 @@ $wc-setting-email-preview-gap: 16px;
 		margin-left: 4px;
 	}
 }
+
+.wc-settings-email-image-url-new-image {
+	border: 1px #ccc solid;
+	border-radius: 4px;
+	width: 400px;
+	height: 88px;
+	position: relative;
+
+	button.wc-settings-email-image-url-select-image {
+		background-color: transparent;
+		border: none;
+		display: block;
+		height: 100%;
+		width: 100%;
+	}
+
+	img.wc-settings-email-image-url-new-image-icon {
+		position: absolute;
+		margin: auto;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
```

Comment:
Just FYI, this could be written as `inset: 0;` ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/inset))